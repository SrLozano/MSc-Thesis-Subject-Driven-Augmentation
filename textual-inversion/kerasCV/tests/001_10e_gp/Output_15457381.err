2023-03-02 15:18:46.453333: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F AVX512_VNNI FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2023-03-02 15:18:47.793641: I tensorflow/core/util/port.cc:104] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2023-03-02 15:18:48.265037: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:18:48.265090: I tensorflow/compiler/xla/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
2023-03-02 15:18:54.394197: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:18:54.396606: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:18:54.396653: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.
2023-03-02 15:19:06.578495: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.580203: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcublas.so.11'; dlerror: libcublas.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.581514: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcublasLt.so.11'; dlerror: libcublasLt.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.582975: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcufft.so.10'; dlerror: libcufft.so.10: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.584306: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcurand.so.10'; dlerror: libcurand.so.10: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.585491: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcusolver.so.11'; dlerror: libcusolver.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.586906: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcusparse.so.11'; dlerror: libcusparse.so.11: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.588176: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudnn.so.8'; dlerror: libcudnn.so.8: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /lsf/10.1/linux3.10-glibc2.17-x86_64/lib:/zhome/d1/6/191852/ENTER/lib/
2023-03-02 15:19:06.588198: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1934] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2023-03-02 15:19:06.588772: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 AVX512F AVX512_VNNI FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
WARNING:tensorflow:From /zhome/d1/6/191852/ENTER/envs/textual-inversion/lib/python3.10/site-packages/tensorflow/python/autograph/pyct/static_analysis/liveness.py:83: Analyzer.lamba_check (from tensorflow.python.autograph.pyct.static_analysis.liveness) is deprecated and will be removed after 2023-09-23.
Instructions for updating:
Lambda fuctions will be no more assumed to be used in the statement where they are used, or at least in the same block. https://github.com/tensorflow/tensorflow/issues/56089
2023-03-02 21:40:14.702329: I tensorflow/compiler/xla/service/service.cc:173] XLA service 0x5ec33c80 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2023-03-02 21:40:14.703355: I tensorflow/compiler/xla/service/service.cc:181]   StreamExecutor device (0): Host, Default Version
2023-03-02 21:40:15.016980: I tensorflow/compiler/mlir/tensorflow/utils/dump_mlir_util.cc:268] disabling MLIR crash reproducer, set env var `MLIR_CRASH_REPRODUCER_DIRECTORY` to enable.
2023-03-02 21:40:18.263332: I tensorflow/compiler/jit/xla_compilation_cache.cc:477] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.
