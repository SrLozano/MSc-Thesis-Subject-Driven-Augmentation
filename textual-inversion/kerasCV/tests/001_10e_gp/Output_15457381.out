Thu Mar  2 15:18:42 2023       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 525.60.13    Driver Version: 525.60.13    CUDA Version: 12.0     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA A100 80G...  Off  | 00000000:CA:00.0 Off |                    0 |
| N/A   27C    P0    40W / 300W |      0MiB / 81920MiB |      0%   E. Process |
|                               |                      |             Disabled |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+
You do not have pycocotools installed, so KerasCV pycoco metrics are not available. Please run `pip install pycocotools`.
You do not have pyococotools installed, so the `PyCOCOCallback` API is not available.
By using this model checkpoint, you acknowledge that its usage is subject to the terms of the CreativeML Open RAIL-M license at https://raw.githubusercontent.com/CompVis/stable-diffusion/main/LICENSE
[[TensorShape([49409, 768])], [], []]
Beggining training

Epoch 1/10
250/250 - 2332s - loss: 0.1355 - 2332s/epoch - 9s/step
Epoch 2/10
250/250 - 2286s - loss: 0.1350 - 2286s/epoch - 9s/step
Epoch 3/10
250/250 - 2288s - loss: 0.1285 - 2288s/epoch - 9s/step
Epoch 4/10
250/250 - 2279s - loss: 0.1383 - 2279s/epoch - 9s/step
Epoch 5/10
250/250 - 2269s - loss: 0.1483 - 2269s/epoch - 9s/step
Epoch 6/10
250/250 - 2266s - loss: 0.1516 - 2266s/epoch - 9s/step
Epoch 7/10
250/250 - 2264s - loss: 0.1286 - 2264s/epoch - 9s/step
Epoch 8/10
250/250 - 2262s - loss: 0.1489 - 2262s/epoch - 9s/step
Epoch 9/10
250/250 - 2263s - loss: 0.1450 - 2263s/epoch - 9s/step
Epoch 10/10
250/250 - 2263s - loss: 0.1324 - 2263s/epoch - 9s/step
 1/50 [..............................] - ETA: 16:37 2/50 [>.............................] - ETA: 11:45 3/50 [>.............................] - ETA: 11:31 4/50 [=>............................] - ETA: 11:16 5/50 [==>...........................] - ETA: 11:03 6/50 [==>...........................] - ETA: 10:47 7/50 [===>..........................] - ETA: 10:33 8/50 [===>..........................] - ETA: 10:20 9/50 [====>.........................] - ETA: 10:0510/50 [=====>........................] - ETA: 9:51 11/50 [=====>........................] - ETA: 9:3612/50 [======>.......................] - ETA: 9:2113/50 [======>.......................] - ETA: 9:0614/50 [=======>......................] - ETA: 8:5115/50 [========>.....................] - ETA: 8:3616/50 [========>.....................] - ETA: 8:2117/50 [=========>....................] - ETA: 8:0618/50 [=========>....................] - ETA: 7:5119/50 [==========>...................] - ETA: 7:3720/50 [===========>..................] - ETA: 7:2221/50 [===========>..................] - ETA: 7:0722/50 [============>.................] - ETA: 6:5223/50 [============>.................] - ETA: 6:3824/50 [=============>................] - ETA: 6:2325/50 [==============>...............] - ETA: 6:0826/50 [==============>...............] - ETA: 5:5427/50 [===============>..............] - ETA: 5:3928/50 [===============>..............] - ETA: 5:2429/50 [================>.............] - ETA: 5:0930/50 [=================>............] - ETA: 4:5531/50 [=================>............] - ETA: 4:4032/50 [==================>...........] - ETA: 4:2533/50 [==================>...........] - ETA: 4:1034/50 [===================>..........] - ETA: 3:5535/50 [====================>.........] - ETA: 3:4136/50 [====================>.........] - ETA: 3:2637/50 [=====================>........] - ETA: 3:1138/50 [=====================>........] - ETA: 2:5739/50 [======================>.......] - ETA: 2:4240/50 [=======================>......] - ETA: 2:2741/50 [=======================>......] - ETA: 2:1242/50 [========================>.....] - ETA: 1:5743/50 [========================>.....] - ETA: 1:4344/50 [=========================>....] - ETA: 1:2845/50 [==========================>...] - ETA: 1:1346/50 [==========================>...] - ETA: 58s 47/50 [===========================>..] - ETA: 44s48/50 [===========================>..] - ETA: 29s49/50 [============================>.] - ETA: 14s50/50 [==============================] - 742s 15s/step
Saving image gandalf...
 1/50 [..............................] - ETA: 12:03 2/50 [>.............................] - ETA: 11:42 3/50 [>.............................] - ETA: 11:32 4/50 [=>............................] - ETA: 11:17 5/50 [==>...........................] - ETA: 11:02 6/50 [==>...........................] - ETA: 10:48 7/50 [===>..........................] - ETA: 10:33 8/50 [===>..........................] - ETA: 10:18 9/50 [====>.........................] - ETA: 10:0410/50 [=====>........................] - ETA: 9:49 11/50 [=====>........................] - ETA: 9:3512/50 [======>.......................] - ETA: 9:2013/50 [======>.......................] - ETA: 9:0514/50 [=======>......................] - ETA: 8:5015/50 [========>.....................] - ETA: 8:3616/50 [========>.....................] - ETA: 8:2117/50 [=========>....................] - ETA: 8:0618/50 [=========>....................] - ETA: 7:5119/50 [==========>...................] - ETA: 7:3620/50 [===========>..................] - ETA: 7:2121/50 [===========>..................] - ETA: 7:0622/50 [============>.................] - ETA: 6:5223/50 [============>.................] - ETA: 6:3724/50 [=============>................] - ETA: 6:2225/50 [==============>...............] - ETA: 6:0826/50 [==============>...............] - ETA: 5:5327/50 [===============>..............] - ETA: 5:3828/50 [===============>..............] - ETA: 5:2429/50 [================>.............] - ETA: 5:0930/50 [=================>............] - ETA: 4:5431/50 [=================>............] - ETA: 4:3932/50 [==================>...........] - ETA: 4:2533/50 [==================>...........] - ETA: 4:1034/50 [===================>..........] - ETA: 3:5535/50 [====================>.........] - ETA: 3:4036/50 [====================>.........] - ETA: 3:2637/50 [=====================>........] - ETA: 3:1138/50 [=====================>........] - ETA: 2:5639/50 [======================>.......] - ETA: 2:4140/50 [=======================>......] - ETA: 2:2741/50 [=======================>......] - ETA: 2:1242/50 [========================>.....] - ETA: 1:5743/50 [========================>.....] - ETA: 1:4344/50 [=========================>....] - ETA: 1:2845/50 [==========================>...] - ETA: 1:1346/50 [==========================>...] - ETA: 58s 47/50 [===========================>..] - ETA: 44s48/50 [===========================>..] - ETA: 29s49/50 [============================>.] - ETA: 14s50/50 [==============================] - 736s 15s/step
Saving image masterpiece...
 1/50 [..............................] - ETA: 12:05 2/50 [>.............................] - ETA: 11:51 3/50 [>.............................] - ETA: 11:37 4/50 [=>............................] - ETA: 11:21 5/50 [==>...........................] - ETA: 11:04 6/50 [==>...........................] - ETA: 10:48 7/50 [===>..........................] - ETA: 10:33 8/50 [===>..........................] - ETA: 10:17 9/50 [====>.........................] - ETA: 10:0310/50 [=====>........................] - ETA: 9:48 11/50 [=====>........................] - ETA: 9:3312/50 [======>.......................] - ETA: 9:1813/50 [======>.......................] - ETA: 9:0414/50 [=======>......................] - ETA: 8:5015/50 [========>.....................] - ETA: 8:3516/50 [========>.....................] - ETA: 8:2017/50 [=========>....................] - ETA: 8:0518/50 [=========>....................] - ETA: 7:5119/50 [==========>...................] - ETA: 7:3620/50 [===========>..................] - ETA: 7:2121/50 [===========>..................] - ETA: 7:0622/50 [============>.................] - ETA: 6:5223/50 [============>.................] - ETA: 6:3724/50 [=============>................] - ETA: 6:2225/50 [==============>...............] - ETA: 6:0826/50 [==============>...............] - ETA: 5:5327/50 [===============>..............] - ETA: 5:3828/50 [===============>..............] - ETA: 5:2329/50 [================>.............] - ETA: 5:0830/50 [=================>............] - ETA: 4:5431/50 [=================>............] - ETA: 4:3932/50 [==================>...........] - ETA: 4:2433/50 [==================>...........] - ETA: 4:1034/50 [===================>..........] - ETA: 3:5535/50 [====================>.........] - ETA: 3:4036/50 [====================>.........] - ETA: 3:2537/50 [=====================>........] - ETA: 3:1138/50 [=====================>........] - ETA: 2:5639/50 [======================>.......] - ETA: 2:4140/50 [=======================>......] - ETA: 2:2741/50 [=======================>......] - ETA: 2:1242/50 [========================>.....] - ETA: 1:5743/50 [========================>.....] - ETA: 1:4344/50 [=========================>....] - ETA: 1:2845/50 [==========================>...] - ETA: 1:1346/50 [==========================>...] - ETA: 58s 47/50 [===========================>..] - ETA: 44s48/50 [===========================>..] - ETA: 29s49/50 [============================>.] - ETA: 14s50/50 [==============================] - 736s 15s/step
Saving image evil...
 1/50 [..............................] - ETA: 12:00 2/50 [>.............................] - ETA: 11:45 3/50 [>.............................] - ETA: 11:30 4/50 [=>............................] - ETA: 11:18 5/50 [==>...........................] - ETA: 11:02 6/50 [==>...........................] - ETA: 10:46 7/50 [===>..........................] - ETA: 10:32 8/50 [===>..........................] - ETA: 10:18 9/50 [====>.........................] - ETA: 10:0310/50 [=====>........................] - ETA: 9:48 11/50 [=====>........................] - ETA: 9:3312/50 [======>.......................] - ETA: 9:1913/50 [======>.......................] - ETA: 9:0414/50 [=======>......................] - ETA: 8:4915/50 [========>.....................] - ETA: 8:3416/50 [========>.....................] - ETA: 8:2017/50 [=========>....................] - ETA: 8:0518/50 [=========>....................] - ETA: 7:5019/50 [==========>...................] - ETA: 7:3620/50 [===========>..................] - ETA: 7:2121/50 [===========>..................] - ETA: 7:0622/50 [============>.................] - ETA: 6:5123/50 [============>.................] - ETA: 6:3624/50 [=============>................] - ETA: 6:2225/50 [==============>...............] - ETA: 6:0726/50 [==============>...............] - ETA: 5:5227/50 [===============>..............] - ETA: 5:3828/50 [===============>..............] - ETA: 5:2329/50 [================>.............] - ETA: 5:0830/50 [=================>............] - ETA: 4:5431/50 [=================>............] - ETA: 4:3932/50 [==================>...........] - ETA: 4:2433/50 [==================>...........] - ETA: 4:1034/50 [===================>..........] - ETA: 3:5535/50 [====================>.........] - ETA: 3:4036/50 [====================>.........] - ETA: 3:2537/50 [=====================>........] - ETA: 3:1138/50 [=====================>........] - ETA: 2:5639/50 [======================>.......] - ETA: 2:4140/50 [=======================>......] - ETA: 2:2741/50 [=======================>......] - ETA: 2:1242/50 [========================>.....] - ETA: 1:5743/50 [========================>.....] - ETA: 1:4244/50 [=========================>....] - ETA: 1:2845/50 [==========================>...] - ETA: 1:1346/50 [==========================>...] - ETA: 58s 47/50 [===========================>..] - ETA: 44s48/50 [===========================>..] - ETA: 29s49/50 [============================>.] - ETA: 14s50/50 [==============================] - 735s 15s/step
Saving image pyramids...
